docs/components/input.md

```diff
@@ -8,6 +8,53 @@
 ```
 
 ## Props
+### Help
+String to be displayed as help text.
+
+- `help`
+
+<KInput help="I can help with that" placeholder="Need help?" />
+
+```
+<KInput help="I can help with that" placeholder="Need help?" />
+```
+
+You also have the option of using the `.help` utility class. This is meant to be used after **KInput** and will be styled appropriately. 
+
+<KInput type="text" placeholder="Need help?" />
+<p class="help">I can help with that</p>
```

Comment:
Non-blocking question: is this supposed to be styled like the input help above?

![Screen Shot 2021-07-26 at 2 08 50 PM](https://user-images.githubusercontent.com/2080476/127059663-1feb29e8-e72d-4b80-b16e-90d2f008f421.png)

Reply:
That's really peculiar I'm not seeing this in the netlify build. Are you seeing this locally? If it's local, you have to run the CSS generator command `yarn build:postcss` to pick up the newly generated styles.

![image](https://user-images.githubusercontent.com/67973710/127180243-d7eb0f97-3bc3-41a0-8c0f-c68cea300c6c.png)

Reply:
Ah thanks! I also had to run `yarn build:styles`, but that seems to have fixed it. Thanks for the tip!

Reply:
Ah, sorry! Yeah that's the one to run, `yarn build:styles` calls the postcss script I mentioned above too!